db/seeds/development.rb

```diff
@@ -27,6 +27,12 @@
 # See doc/COPYRIGHT.rdoc for more details.
 #++
 
+# Disable mail delivery for the duration of this task
+ActionMailer::Base.perform_deliveries = false
+
+# Avoid asynchronous DeliverWorkPackageCreatedJob
+Delayed::Worker.delay_jobs = false
```

Comment:
We could probably make this more explicit by disabling work package notifications jobs (using the Setting), but I argue that there is not much harm in this line anyway, it simply allows us to control all settings in the context of this script, without later asynchronous execution.